document/docs/chart-properties/scroll-mode.md

```
@@ -0,0 +1,86 @@
+|------|------------|---------|-------------|
+| `ScrollMode.Disabled` | None | Default | Disables scrolling. All bars are fitted within the visible chart area. Drag gestures trigger popups. |
+| `ScrollMode.Auto` | `maxBarSpacing: Dp` | `16.dp` | Automatically enables scrolling if bars become too compressed in the available viewport. |
+| `ScrollMode.Always` | `barSpacing: Dp` | `16.dp` | Always enables scrolling with a fixed empty gap (`barSpacing`) between adjacent bars. |
+
+---
+
+## How `barSpacing` Works
+
+The `barSpacing` parameter represents the **empty gap between two adjacent bars**, not the distance between their origins.
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)